azure-pipelines.yml

```diff
@@ -11,7 +11,7 @@ trigger:
     include:
     - v4.x
     - release_4.0
-
+ 
```

Comment:
```suggestion

```

Reply:
The new line is fine, but should remove empty space at very least. Or undo all changes to this file!